Makefile

```diff
@@ -42,6 +47,9 @@ run-blinky2: build/blinky2
 ifeq ($(TARGET),pca10040)
 flash-%: build/%.hex
 	nrfjprog -f nrf52 --sectorerase --program $< --reset
+else ifeq ($(TARGET),microbit)
+flash-%: build/%.hex
+	nrfjprog -f nrf51 --sectorerase --program $< --reset
```

Comment:
Have you tried using openocd with `-f interface/cmsis-dap.cfg`?
In my experience, nrfjprog only works when the chip has Segger firmware which is non-standard.

Reply:
It's in microbit.json:

    openocd -f interface/cmsis-dap.cfg -f target/nrf51.cfg -c 'program {hex} reset exit'

Translated to Makefile syntax (see bluepill/stm32):

    openocd -f interface/cmsis-dap.cfg -f target/nrf51.cfg -c 'program $< reset exit'

Reply:
That works great, and no firmware update required is better. Still seems like adding to the makefile for easy loading would be a good thing. Should I update this PR accordingly?

Reply:
Yes, please. Then I'm sure it works for both of the firmwares :)